Dockerfile.local

```
@@ -1,4 +1,4 @@
-FROM golang:1.24.1-bullseye AS base
+FROM golang:1.24.6-bookworm AS base
```

Local dockerfile does not need to be bookworm, only CI ones.
```suggestion
FROM golang:1.24.6 AS base
```